nixos/modules/services/hardware/handheld-daemon.nix

```diff
@@ -31,6 +40,11 @@ in
 
       restartIfChanged = true;
 
+      path = mkIf cfg.ui.enable [
+        pkgs.handheld-daemon-ui
```

Comment:
Maybe this should be wrapped inside the UI?

Reply:
Accidentally started a review instead of commenting so I'll post my comment again:

I'm not sure what you mean?

Handheld-daemon needs the ui to be in it's PATH, so I would need to wrap handheld-daemon, not the ui